src/OpenSpaceToolkit/Astrodynamics/EventCondition/DurationCondition.cpp

```diff
@@ -10,22 +10,21 @@ namespace eventcondition
 {
 
 DurationCondition::DurationCondition(const Criteria& aCriteria, const Duration& aDuration)
-    : EventCondition("Duration Condition", aCriteria),
-      durationInSeconds_(aDuration.inSeconds())
+    : EventCondition("Duration Condition", aCriteria, aDuration.inSeconds())
 {
 }
 
 DurationCondition::~DurationCondition() {}
 
 Duration DurationCondition::getDuration() const
 {
-    return Duration::Seconds(durationInSeconds_);
+    return Duration::Seconds(getTarget());
 }
 
-Real DurationCondition::evaluate(const VectorXd& aStateVector, const Real& aTime) const
+Real DurationCondition::compute(const VectorXd& aStateVector, const Real& aTime) const
 {
     (void)aStateVector;
-    return aTime - durationInSeconds_;
+    return aTime;
```

Comment:
Alright, should all be backward compatible as `evaluate` remains the method computing the difference between target and value, confirmed?